heatmap.py

```diff
@@ -658,14 +662,13 @@ def create_movie(infiles, outfile, ffmpegopts):
         logging.info('Encoding video: %s' % ' '.join(command))
         subprocess.call(command)
 
-
     def run(self):
         logging.info('Putting animation frames in %s' % self.tmpdir)
         self.inputs_since_output = 0
         self.frame_count = 0
         matrix = process_shapes(self.config, self.maybe_save_image)
-        if ( not self.frame_count
-             or self.inputs_since_output >= self.config.straggler_threshold ):
+        if (not self.frame_count or
+                self.inputs_since_output >= self.config.straggler_threshold):
```

Comment:
Ah, I see.  I read up on that error and the relevant PEP8 passages.  The original indentation is explicitly permitted (despite E129), but I can see how it's potentially confusing. I think the above indentation is also confusing, but I can change it in a later commit.  Thanks.  I will add pep8 conformance to my pre-commit test suite.  :)